controller/deliveryEmployeeController.ts

```diff
@@ -21,6 +21,20 @@ module.exports = function(app: Application) {
         res.render('list-delivery-employees', { employees: data });
     })
 
+    app.get('/deliveryemployees/:id', async (req: Request, res: Response) => {
+        let data: DeliveryEmployee = new DeliveryEmployee;
```

Comment:
Should this not be `let data: DeliveryEmployee`?

Reply:
It wasn't liking this